news/2021-02-04-modding-mentorship-cycle-2-signups-now-open

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Modding Mentorship Cycle 2: Signups Now Open
+date: 2021-02-04 10:00:00 +0000
+---
+
+Greetings, modders! With the new year starting, we want to announce the second run of Modding Mentorship! If you ever wanted to polish your modding abilities or assist newer members of modding communities in their way to achieve the needed modding skills, this is your time to shine!
```

Comment:
```suggestion
Greetings, modders! With the new year starting, we want to announce the second run of Modding Mentorship! If you ever wanted to polish your modding abilities or assist novice modders in their way to achieving the needed modding skills, this is your time to shine!
```
the sentence sounds a little weird when read carefully, you might want to simplify it the way I suggested